ajax/libs/simplebar/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "simplebar",
+  "filename": "simplebar.js",
+  "description": "Scrollbars, simpler.",
+  "homepage": "https://grsmto.github.io/simplebar/",
+  "keywords": [
+    "simplebar",
+    "scrollbar"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "http://github.com/grsmto/simplebar.git"
+  },
+  "author": "Adrien Grsmto from a fork by Jonathan Nicol",
+  "license": "MIT",
+  "npmName": "simplebar",
+  "npmFileMap": [
```

Comment:
Let's use git auto-update because there are more versions from GitHub repo (32) then npm (19).
We host more versions can let web front-end developer more choose.
Thank you.